src/elements/Input/Input.less

```diff
@@ -3,19 +3,21 @@
 	// Checkbox Styling
 	&[type="checkbox"] {
 		opacity: 0; // Hide default
+		width: 0;
+		height: 0;
 
 		& + label:before {
 			position: relative;
 			top: 6px;
 			content: '';
 			margin-right: 10px;
 			display: inline-block;
-			width: 20px;
-			height: 20px;
+			width: 16px;
```

Comment:
do we want to use the UI vars (ui-small) for height/width?

Reply:
Yep, there's a number of small positioning/sizing changes that need to be made to a number of our components. I'll take care of them in that cleanup I was talking about earlier.